fvtest/compilertest/README.md

```diff
@@ -2,42 +2,66 @@ Test Compiler
 =============
 
 This file contains the Eclipse OMR 'Test compiler', a framework for building
-language independent compiler tests for the Testarossa compiler technology. 
+language independent compiler tests for the Testarossa compiler technology.
 
 Building
 --------
 Before building the Test compiler you must make sure to configure the project
-by following the 'Basic configuration and compile' step as outlined in the 
-top-level `omr/README.md` document.
+by following the 'Basic configuration and compile' step as outlined in the
+top-level `<omr_root_folder>/README.md` document.
 
 To build the Test compiler, typing `make` in this directory
-(`omr/fvtest/compilertest`) should suffice. A binary called 
+(`<omr_root_folder>/fvtest/compilertest`) should suffice. A binary called
 `testjit` will be produced in the top-level `omr` directory.
 
-BUILD_CONFIG defaults to prod. 
+BUILD_CONFIG defaults to prod.
 
 In the case where `guess-platform.sh` is incorrect, you can set the `PLATFORM`
 environment variables to one of the supported ones, listed in
-`compilertest/build/platforms/host.mk`. 
+`<omr_root_folder>/fvtest/compilertest/build/platforms/host.mk`.
 
 
 Running the Test Compiler
 -------------------------
 
-The Test compiler has been written using [Google Test][gtest], and as such, 
-obeys the Google Test command line arguments, that can be seen by running 
-`testjit --help`. 
+The Test compiler has been written using [Google Test][gtest], and as such,
+obeys the Google Test command line arguments, that can be seen by running
+`testjit --help`.
 
-A very useful subset include: 
+A very useful subset include:
 
-* `--gtest_list_tests` Lists the tests
-* `--gtest_filter=` Filters what runs based on a simple regex. 
+* `--gtest_list_tests` Lists the tests.
+* `--gtest_filter=` Filters what runs based on a simple regex.
+* `--gtest_also_run_disabled_tests` Also run disabled test group.
 
 Options can be passed to the Testarossa compiler technology using the
 `TR_Options` environment variable.
 
     $ TR_Options='log=/tmp/testcomplog,tracefull' testjit
 
 
+Contributing to Test Compiler
+-----------------------------
 
-[gtest]: https://github.com/google/googletest/ 
+The Test compiler uses ILInjectors to directly invoke code generator
```

Comment:
This statement is not quite correct. It uses IlInjectors to directly generate simple IL designed to target particular code generator evaluators. The code generator evaluators are not "directly invoked".